### ISSUE TITLE
XML Frames

### ISSUE THREAD

**Issue by liberlog** [bpmnmatrix/bpmnmatrix.github.io#34]
XML Frames is a Free BPMN W4 EXPRESS Engine on Pascal language.
http://xmlframes.com/

**Comment by moritzhesse** [comment]
Thank you for your contribution! Could you please help by filling out the "New application" template?

**Name:** 
**Version:** (Application Version)
**URL:** (Full URL)
**Creator:** (Company/Author)

## Platform/OS

**Platform/OS:** (Describe supported OSs)
**Cross-Platform:** yes/no
**Windows:** yes/no
**Mac:** yes/no
**Linux:** yes/no
**Unix:** yes/no

## Versions/Releases

**BPMN Version:** 
**First Release:** (Date)
**Latest Release:** (Date)

## Licensing

**License free:** yes/no
**Proprietary:** yes/no
**Shareware:** yes/no
**Open Source:** yes/no

**Comment by Mat-Hac** [comment]
{
"Name": "XML Frames",
"Version": "",
"URL": "https://xmlframes.com/",
"Creator": "Matthieu Giroux",
"Platform/OS": "Client/Server",
"OS": {
"Cross-Platform": true,
"Windows": true,
"Mac": it was,
"Linux": true,
"Unix": it can be
},
"BPMN Version": "Depends on W4 BPMN+",
"First Release": "2009",
"Latest Release": "2021",
"License": {
"free": true,
"proprietary": false,
"Shareware": false,
"Open Source": true
}
}

**Comment by moritzhesse** [comment]
Duplicate of #54.
